platform/view/services/kvs/kvs_test.go

```diff
@@ -125,7 +125,7 @@ func testRound(t *testing.T, cfg *fakeProv) {
 
 	for ctr := 0; it.HasNext(); ctr++ {
 		val = &stuff{}
-		err = it.Next(val)
+		_, err = it.Next(val)
```

Comment:
can you please add a test case to check that we get the expected key as well?!

Reply:
done